client/src/app/api/rest.ts

```diff
@@ -314,6 +315,22 @@ export const getApplications = (): Promise<Application[]> =>
 export const updateApplication = (obj: Application): Promise<Application> =>
   axios.put(`${APPLICATIONS}/${obj.id}`, obj);
 
+export const getApplicationAnalysis = (
+  applicationId: number,
+  type: MimeType
+): Promise<void> =>
+  axios.get(
+    `${APPLICATIONS}/${String(applicationId)}/analysis${
+      type === MimeType.TAR ? "/report" : ""
+    }`,
+    {
+      responseType: "blob",
+      headers: {
```

Comment:
No accept headers are required for the /report request any longer as I mentioned https://github.com/konveyor/tackle2-ui/pull/1329 . I am not really following the intent of this PR. I think effort would be better spent on one of our open kanban issues. Thx.